x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -51,7 +51,7 @@ export const ANCESTRY_LIMIT: number = 2;
 
 const Windows: OSFields[] = [
   {
-    name: 'windows 10.0',
+    name: 'Windows',
```

Comment:
Update the `name` field for OS in the generator to reflect reality from the Endpoint: https://github.com/elastic/endpoint-dev/blob/d233f42dde60a05fbfbd194695236c0c09341aa7/Elastic/Libraries/ECS/Lib/Constants.cpp#L192-L201